app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -16,7 +17,7 @@
         ].select(&:present?).join(' '),
       ) do %>
     <%= image_tag(asset_url("mfa-options/#{option.type}.svg"), alt: "#{option.label} icon", class: 'usa-checkbox__image') %>
-    <div class="usa-checkbox__label--text"><%= option.label %>
+    <div class="usa-checkbox__label--text"><%= option.label %> <%= option.mfa_configuration %>
```

Comment:
WDYT of renaming this method to `mfa_configuration_description` or something? Based on the name of the method it sounds like a type or something not user-facing